lib/utilities/size_config.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/widgets.dart';
+
+class SizeConfig {
+  static MediaQueryData _mediaQueryData;
+  static double screenWidth;
+  static double screenHeight;
+  static double defaultSize;
+  static Orientation orientation;
+
+  SizeConfig(BuildContext context) {
+    _mediaQueryData = MediaQuery.of(context);
+    screenWidth = _mediaQueryData.size.width;
+    screenHeight = _mediaQueryData.size.height;
+    orientation = _mediaQueryData.orientation;
+    defaultSize = screenWidth * 0.024;
```

Comment:
What is the reason that this multiplier is for default size?

Reply:
So I used this in a previous flutter project. The idea is that defaultSize is used as a multiplier for all sizes so, for example, all paddings would be written in the format of `Constants.defaultSize * x` where x is whatever number we need.

I'm considering taking this off TBH since this is now only being used in `login_page.dart` which is already completely refactored on my following PR. So rest assure that this will be gone in my next PR since `login_page.dart` will also be gone.

I'm intending to keep the `screenWidth`, height and orientation tho.